src/Module.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Mvc\View;
+
+use function array_merge;
+
+final class Module
+{
+    public function getConfig(): array
+    {
+        $provider = new ConfigProvider();
+        $config   = $provider();
+        unset($config['dependencies']);
+
+        return array_merge([
+            'service_manager' => $provider->getDependencyConfig(),
```

Comment:
Is this the only reason for having the `getDependencyConfig()` method? If so, this could just become `return $config;` until such time as we need to expand it.

Reply:
I guess the reason I must've done this is to nail down swapping the array key from `dependencies` to `service_manager` - agree it's not necessary for this patch. Happy to remove both here and in `ConfigProvider` if required?